app/templates/components/donations/donation-status.hbs

```diff
@@ -0,0 +1,7 @@
+{{#if subscription}}
+  {{donations/show-donation amount=subscription.amount}}
+{{else if processStarted}}
+  {{donations/create-donation subscription=subscription continue=(action createDonation) amount=15}}
```

Comment:
Good catch, It's a remnant from the start of this PR, when the implementation was different.